tests/sentry/api/endpoints/test_integrations_slack_commands.py

```diff
@@ -52,6 +63,37 @@ def test_method_get_not_allowed(self):
 class SlackCommandsPostTest(SlackCommandsTest):
     method = "post"
 
+    def setUp(self):
```

Comment:
You should split `SlackCommandsPostTest` into `SlackCommandsHelpTest` and `SlackCommandLinkTest`. The existing tests don't need the extra steps in `.setUp()`.